python/cudf/cudf/_lib/scalar.pyx

```diff
@@ -34,17 +34,23 @@ from cudf._lib.cpp.wrappers.durations cimport(
     duration_us,
     duration_ns
 )
+from cudf._lib.cpp.wrappers.decimals cimport decimal64, scale_type
 from cudf._lib.cpp.scalar.scalar cimport (
     scalar,
     numeric_scalar,
     timestamp_scalar,
     duration_scalar,
-    string_scalar
+    string_scalar,
+    fixed_point_scalar
 )
+from cudf.utils.dtypes import _decimal_to_int64
 cimport cudf._lib.cpp.types as libcudf_types
 
+
 cdef class DeviceScalar:
 
+    dtype_metadata = {}
```

Comment:
Do we expect to hold any metadata other than things related to the dtype? If not, maybe we should just create a `_dtype` object during construction, and then use that as needed.